### PR TITLE
fixed sonar issues

### PR DESCRIPTION
**Pull request by mfromme**

### CODE REVIEW

File: services/ppac/src/test/java/app/coronawarn/datadonation/services/ppac/ios/verification/scenario/ratelimit/ProdPpacIosRateLimitStrategyTest.java [hunk 1]
```diff
@@ -115,11 +115,6 @@ void shouldNotThrowExceptionWhenValidateForPpaIsMoreThan23HoursSameDay() {
     TimeUtils.setNow(null);
   }
 
-  @Test
-  void testAllRelevantSrsCases() {
-    //FIXME: Implement tests for SRS
```
Comment (mfromme) [inline services/ppac/src/test/java/app/coronawarn/datadonation/services/ppac/ios/verification/scenario/ratelimit/ProdPpacIosRateLimitStrategyTest.java:120]: the SRS tests are already implemented

Comment (roesslerj) [reply services/ppac/src/test/java/app/coronawarn/datadonation/services/ppac/ios/verification/scenario/ratelimit/ProdPpacIosRateLimitStrategyTest.java:120]: Where are they implemented? If so, we should just remove this method...

File: services/ppac/src/main/java/app/coronawarn/datadonation/services/ppac/android/controller/TestProfileExceptionHandler.java [hunk 2]
```diff
@@ -14,11 +14,11 @@
...
 public class TestProfileExceptionHandler extends ResponseEntityExceptionHandler {
 
-  private static final Logger LOGGER = LoggerFactory.getLogger(TestProfileExceptionHandler.class);
+  private static final Logger logger = LoggerFactory.getLogger(TestProfileExceptionHandler.class);
 
   @ExceptionHandler(Exception.class)
   @ResponseStatus(HttpStatus.INTERNAL_SERVER_ERROR)
   public void unknownException(final Exception ex, final WebRequest wr) {
-    LOGGER.error("Unable to handle " + wr.getDescription(false), ex);
+    logger.error("Unable to handle " + wr.getDescription(false), ex);
```
[the remark this answers is not in the document]
Comment (roesslerj) [reply services/ppac/src/main/java/app/coronawarn/datadonation/services/ppac/android/controller/TestProfileExceptionHandler.java:22]: The Log Injection concern looks legit... is it not?

**Review by roesslerj — DISMISSED** [review]
And there seems to be a test failing...

File: services/ppac/src/main/java/app/coronawarn/datadonation/services/ppac/android/controller/TestProfileExceptionHandler.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (mfromme) [reply services/ppac/src/main/java/app/coronawarn/datadonation/services/ppac/android/controller/TestProfileExceptionHandler.java:22]: This was a sonar issue that stated 'the superclass also has a variable called 'LOGGER', so it's best to rename this one. I will make another change to fix this.

File: services/ppac/src/test/java/app/coronawarn/datadonation/services/ppac/ios/verification/scenario/ratelimit/ProdPpacIosRateLimitStrategyTest.java (hunk 1 above)
Comment (mfromme) [reply services/ppac/src/test/java/app/coronawarn/datadonation/services/ppac/ios/verification/scenario/ratelimit/ProdPpacIosRateLimitStrategyTest.java:120]: I removed this method exactly for that reason already. The implementation of the SRS use cases is already in the same test class:
* shouldThrowExceptionWhenValidateForSrsIsOnTheSameWeek
* shouldNotThrowExceptionWhenOnlyUpdateWeekIsTheSameSrs

**Comment by mfromme** [comment]
> And there seems to be a test failing...

This is very strange. The test validates the id that is persisted to DB. And the resulting fetch gets the ID, but it has lots of trailing white space. I have to investigate.

**Review by hilmarf — APPROVED** [review]

**Pull request merged**